src/main/java/muramasa/antimatter/item/ItemBasic.java

```diff
@@ -71,10 +71,6 @@ public boolean isEqual(ItemStack stack) {
         return stack.getItem() == this;
     }
 
-    public static boolean doesShowExtendedHighlight(ItemStack stack) {
-        return AntimatterAPI.getCoverFromCatalyst(stack) != null;
-    }
-
```

Comment:
Comment this out instead of removing it

Reply:
What it is even for?

Reply:
Remnant code from 1.12 where extended highlighting can be shown under certain situations. Comment it out, add a TODO so someone can reimplement it

Reply:
ok